cfgov/core/tests/test_management_inactive_users.py

```diff
@@ -0,0 +1,110 @@
+        self.user_3 = User.objects.create(username='üser_3',
+                                          last_login=days_89,
+                                          date_joined=days_91)
+
+        # This user has never logged in, joined 91 days ago
+        self.user_4 = User.objects.create(username='user_4',
+                                          date_joined=days_91)
+
+        # This user has never logged in, joined today.
+        self.user_5 = User.objects.create(username='user_5')
+
+        self.stdout = StringIO()
+
+    def get_stdout(self):
+        return self.stdout.getvalue().decode('utf-8')
+
+    def test_format_inactive_users_last_login(self):
+        short_date = date_format(self.user_1.last_login,
+                                 "SHORT_DATETIME_FORMAT")
+        self.assertEqual(
+            Command().format_inactive_users([self.user_1]),
+            '\tuser_1: {}\n'.format(short_date)
+        )
+
+    def test_format_inactive_users_never_logged_in(self):
+        self.assertEqual(
+            Command().format_inactive_users([self.user_5]),
+            '\tuser_5: never\n'
+        )
+
+    def test_get_inactive_users(self):
+        """ Test that two users are listed for the default 90 period
+        including one that last logged in 90 days ago. """
+        call_command('inactive_users', stdout=self.stdout)
+        self.assertIn("user_1", self.get_stdout())
+        self.assertIn("user_2", self.get_stdout())
+        self.assertIn("user_4", self.get_stdout())
+        self.assertNotIn("üser_3", self.get_stdout())
+        self.assertNotIn("user_5", self.get_stdout())
+
+    def test_get_inactive_users_87_days(self):
+        """ Test that all users are listed for a custom 87 day period """
+        call_command('inactive_users', period=87, stdout=self.stdout)
+        self.assertIn("user_1", self.get_stdout())
+        self.assertIn("user_2", self.get_stdout())
+        self.assertIn("üser_3", self.get_stdout())
+        self.assertIn("user_4", self.get_stdout())
+        self.assertNotIn("user_5", self.get_stdout())
+
+    def test_get_inactive_users_92_days(self):
+        """ Test that no users are listed for a custom 92 day period """
+        call_command('inactive_users', period=92, stdout=self.stdout)
+        self.assertNotIn("user_1", self.get_stdout())
+        self.assertNotIn("user_2", self.get_stdout())
+        self.assertNotIn("üser_3", self.get_stdout())
+        self.assertNotIn("user_4", self.get_stdout())
+        self.assertNotIn("user_5", self.get_stdout())
+
+    @mock.patch('core.management.commands.inactive_users.mail.EmailMessage')
+    def test_sends_email(self, mock_EmailMessage):
+        """ Test that mail.EmailMessage is called with the appropriate
+        list of users """
+        call_command('inactive_users',
+                     emails=['test@example.com'],
+                     stdout=self.stdout)
+        mock_EmailMessage.return_value.send.assert_called_once()
```

Comment:
There's a nicer way to do this without mocks, if you're so inclined. During tests that use the `locmem` backend (which we do, [in our test settings](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/cfgov/settings/test.py#L10)), Django provides [a test outbox](https://docs.djangoproject.com/en/1.8/topics/testing/tools/#django.core.mail.django.core.mail.outbox) that you can check against without needing to mock anything. See an example test from our repo that uses this [here](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/v1/tests/test_email.py#L56).